src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/TimeBasedQuery/TimeBasedQuery.xaml.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Data;
+using System;
+using Esri.ArcGISRuntime;
+
+namespace ArcGISRuntime.WPF.Samples.TimeBasedQuery
+{
+    public partial class TimeBasedQuery
+    {
+        // Hold a URI pointing to the feature service
+        private Uri _serviceUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0");
+
+        // Hold a reference to the feature table used by the sample
+        private ServiceFeatureTable _myFeatureTable;
+
+        public TimeBasedQuery()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create a new map with oceans basemap
+            Map myMap = new Map(Basemap.CreateOceans());
+
+            // Create feature table for the incident feature service
+            _myFeatureTable = new ServiceFeatureTable(_serviceUri);
+
+            // Define the request mode
+            _myFeatureTable.FeatureRequestMode = FeatureRequestMode.ManualCache;
+
+            // When feature table is loaded, populate data
+            _myFeatureTable.LoadStatusChanged += OnLoadedPopulateData;
+
+            // Create FeatureLayer that uses the created table
+            FeatureLayer myFeatureLayer = new FeatureLayer(_myFeatureTable);
+
+            // Add created layer to the map
+            myMap.OperationalLayers.Add(myFeatureLayer);
+
+            // Assign the Map to the MapView
+            MyMapView.Map = myMap;
+        }
+
+        private async void OnLoadedPopulateData(object sender, LoadStatusEventArgs e)
+        {
+            // If layer isn't loaded, do nothing
+            if (e.Status != LoadStatus.Loaded) { return; }
+
+            // Create new query object that contains parameters to query specific request types
```

Comment:
I don't understand this comment. Is it specific to the "incidents" layer that was originally used (what are "request types", i.e.)? I think something like "Create a new query parameters object" would suffice.